store/conf/attrs/zimbra-attrs.xml

```diff
@@ -10560,4 +10560,14 @@ TODO: delete them permanently from here
   <defaultCOSValue>FALSE</defaultCOSValue>
   <desc>Feature to enable delivery status notification</desc>
 </attr>
+
+<attr id="4136" name="zimbraFeatureExternalEmailWarningEnabled" type="boolean" cardinality="single" optionalIn="globalConfig,domain" flags="domainInherited" since="10.1.7">
+  <globalConfigValue>FALSE</globalConfigValue>
+  <desc>Feature to enable/disable External Email Warning (EEW)</desc>
+</attr>
+
+<attr id="4137" name="zimbraExternalEmailWarningMessage" type="string" cardinality="single" optionalIn="globalConfig,domain" flags="domainInherited" since="10.1.7">
+  <desc>Sets the base warning text for External Email Warning (EEW)</desc>
+  <globalConfigValue>This message originated outside of your organization.</globalConfigValue>
+</attr>
```

Comment:
Please change IDs. 4136 and 4137 are reserved.
https://github.com/Zimbra/zm-mailbox-zok/pull/693

Reply:
@k-kato The one that will be merged first into develop will use the ids, other will have update, rebase and then merge.